build-tools/packages/build-cli/src/commands/check/policy.ts

```diff
@@ -228,16 +235,15 @@ export class CheckPolicy extends BaseCommand<typeof CheckPolicy.flags> {
         }
 
         CheckPolicy.count++;
-        if (exclusions.every((value) => !value.test(line))) {
+        if (!exclusions.every((value) => !value.test(line))) {
             this.log(`Excluded: ${line}`);
             return;
         }
 
         try {
             this.routeToHandlers(filePath, handlerRegex);
-            runPolicyCheck(this.processedFlags.fix);
```

Comment:
Just double checking you want to delete this.

Reply:
Yes, it’s redundant. Every handleLine call is wrapped around a try finally block that calls runPolicyCheck